Eask

```diff
@@ -0,0 +1,20 @@
+(package "async"
+         "1.9.7"
+         "Asynchronous processing in Emacs")
+
+(website-url "https://github.com/jwiegley/emacs-async")
+(keywords "async")
+
+(package-file "async.el")
+
+(files "async.el" "async-bytecomp.el" "*-async.el")
+
+(script "test" "echo \"Error: no test specified\" && exit 1")
+
+(source "gnu")
+(source "melpa")
```

Comment:
GNU ELPA packages are expected to avoid dependencies on MELPA.
What is this needed for?  If it's `buttercup`, then it is also distributed via NonGNU ELPA: https://elpa.nongnu.org/nongnu/buttercup.html
Although I always forget whether it's okay for GNU ELPA to depend on NonGNU ELPA (maybe someone else remembers?).

Reply:
It doesn't depend on it, only for tests.  Those can be safely ignored, since they are not packaged, no?

I see it as a dependency for development only.

Reply:
> It doesn't depend on it, only for tests. Those can be safely ignored, since they are not packaged, no?

FWIW `async-test.el` is currently packaged.

Reply:
Yea, but it's not really a test :sweat_smile: More like a cookbook.